test/component-test.js

```diff
@@ -59,7 +59,20 @@ describe('component creation', () => {
     });
 
     context('node is not scoped', () => {
-      it('recursively creates elements with data');
+      it('recursively creates elements with data', () => {
+        var chattr = new Attribute('data-prop', 'wat');
+        var child = new Node('div', [chattr]);
+        var node = new Node('div', [], [child]);
+        var data = {wat: 'hahaha'};
```

Comment:
As much as this is a pretty trivial test case, I think there could be some documentation value in naming the key and value something a bit more analogous to a real-world scenario, to help things sink in at a glance. Maybe `{name: 'Aloysius'}`, etc.